### ISSUE TITLE
IAM spreg Tool Expects eID to be a valid url

### ISSUE THREAD

**Issue by JoA-MoS** [drstearns/passport-uwshib#6]
when using the auto detect feature of IAM Spreg that checks /Shibboleth.sso/Metadata route the eID is expected to be a valid URL, meaning it needs to have "https://" in front of the domain.  The example server excludes this but can easily be changed in the example server or could be addressed in the strategy.  

Line 66 of the example server:
```
var strategy = new uwshib.Strategy({
    //Updated entity Id for UW spreg tool
    entityId: 'https://' + domain,
    privateKey: privateKey,
    callbackUrl: loginCallbackUrl,
    domain: domain
    //If your server is not using time.u.washington.edu
    //as its authoritative time server (including if your
    //server is running within the NetID domain!), add
    //the following property setting.
    //This will allow for a small amount of
    //skew between the clocks of the client and the server.
    //
    //acceptedClockSkewMs: 200
});
```

**Comment by drstearns** [comment]
I'm a little reluctant to change the behavior of the strategy object at this point. I don't want to break other users who may be relying on the current behavior. Are you OK with me simply adding `https://` to the domain name in the example server script, along with a comment about why that's necessary for the UW spreg tool? Or do you think the strategy (and thus default behavior) needs to be changed?

**Comment by JoA-MoS** [comment]
I understand the concern with breaking anyone else who is already using the code as is.  I am fine with just updating the example server and comments for future users who run into this issue.  

In addition I have created an example server using this library that runs in an Azure web app.  There are some changes that need to be made to the server for it to work on iisnode of Azure and how to create your https/http server (specifically you don't).  If you would like I could include a separate example server with readme.md for azure.

Otherwise everything worked smoother than I had expected.  Thank You!

**Comment by drstearns** [comment]
OK, I added the `https://` prefix to the domain name in the example script, along with a comment as to why. Fixed in commit e8a222f.
